flatland_plugins/include/flatland_plugins/gps.h

```diff
@@ -0,0 +1,82 @@
+#include <flatland_plugins/update_timer.h>
+#include <flatland_server/model_plugin.h>
+#include <flatland_server/timekeeper.h>
+#include <flatland_server/types.h>
+#include <ros/ros.h>
+#include <sensor_msgs/NavSatFix.h>
```

Comment:
I think it should technically already have been there due to the laser scan plugin, but we're missing a build/run dependency on sensor_msgs in package.xml for flatland_plugins.